### PR TITLE
zh: Refine prerequisites.md

### PR DESCRIPTION
**Pull request by weekface**
### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

**Comment by Ishiihara** [comment]
LGTM

File: zh/prerequisites.md [hunk 1]
```diff
@@ -16,188 +16,182 @@ category: how-to
...
 
-如果没有加载，则执行如下命令加载：
+## 配置 Iptables
 
-{{< copyable "shell-regular" >}}
+FORWARD 链默认配置成 ACCEPT：
 
 ```shell
-modprobe br_netfilter
+iptables -P FORWARD ACCEPT
```
Comment (DanielZhangQD) [inline zh/prerequisites.md:35]: Can this rule be preserved after the node reboots?

File: zh/prerequisites.md [hunk 2]
```diff
@@ -16,188 +16,182 @@ category: how-to
...
-以上命令输出的第一列是中断号，第二列是设备名称。如果是多队列网卡，上面的命令会显示多行信息，网卡的每个队列对应一个中断号。通过以下命令可以查看该中断号被绑定到哪个 CPU 上：
+## 配置 Irqbalance 服务
+
+[Irqbalance](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/6/html/performance_tuning_guide/sect-red_hat_enterprise_linux-performance_tuning_guide-tool_reference-irqbalance) 服务可以将各个设备对应的中断号分别绑定到不同的 CPU 上，以防止所有中断请求都落在同一个 CPU 上而引发性能瓶颈。
 
 {{< copyable "shell-regular" >}}
 
 ```shell
-cat /proc/irq/${ir_num}/smp_affinity
+systemctl start irqbalance
```
Comment (DanielZhangQD) [inline zh/prerequisites.md:94]: 需要 `systemctl enable irqbalance` 吗？

File: zh/prerequisites.md [hunk 3]
```diff
@@ -16,188 +16,182 @@ category: how-to
...
-* 设置 Docker 服务的 `ulimit`
+2. 设置 Docker daemon 的 ulimit： 
 
     {{< copyable "shell-regular" >}}
-
+    
     ```shell
-    sudo vim /etc/systemd/system/docker.service
+    vim /etc/systemd/system/docker.service
+    LimitNOFILE=1048576
```
Comment (DanielZhangQD) [inline zh/prerequisites.md:154]: `LimitNPROC` 这个需要修改不？

Comment (weekface) [reply zh/prerequisites.md:154]: 这个不需要

File: zh/prerequisites.md (hunk 1 above)
Comment (weekface) [reply zh/prerequisites.md:35]: 添加了一条提示，加入到开机启动脚本

File: zh/prerequisites.md (hunk 2 above)
Comment (weekface) [reply zh/prerequisites.md:95]: done

File: zh/prerequisites.md [hunk 4]
```diff
@@ -16,188 +16,182 @@ category: how-to
...
     > **注意：**
     >
     > `LimitNOFILE` 需要显式设置为 `1048576` 或者更大，而不是默认的 `infinity`，由于 `systemd` 的 [bug](https://github.com/systemd/systemd/commit/6385cb31ef443be3e0d6da5ea62a267a49174688#diff-108b33cf1bd0765d116dd401376ca356L1186)，`infinity` 在 `systemd` 某些版本中指的是 `65536`。
 
-## 硬件和部署要求
-
-与使用 binary 方式部署 TiDB 集群一致，要求选用 Intel x86-64 架构的 64 位通用硬件服务器，使用万兆网卡。关于 TiDB 集群在物理机上的具体部署需求，参考 [TiDB 软件和硬件环境建议配置](https://pingcap.com/docs-cn/v3.0/how-to/deploy/hardware-recommendations)。
-
-对于服务器 disk、memory、CPU 的选择要根据对集群的容量规划以及部署拓扑来定。线上 Kubernetes 集群部署为了保证高可用，一般需要部署三个 master 节点、三个 etcd 节点以及若干个 worker 节点。同时，为了充分利用机器资源，master 节点一般也充当 worker 节点（也就是 master 节点上也可以调度负载）。通过 kubelet 设置[预留资源](https://kubernetes.io/docs/tasks/administer-cluster/reserve-compute-resources/)来保证机器上的系统进程以及 Kubernetes 的核心进程在工作负载很高的情况下仍然有足够的资源来运行，从而保证整个系统的稳定。
+## K8s 服务
```
Comment (ran-huang) [inline zh/prerequisites.md:164]:
```suggestion
## Kubernetes 服务
```

File: zh/prerequisites.md [hunk 5]
```diff
@@ -16,188 +16,182 @@ category: how-to
...
 
-下面按 3 Kubernetes master + 3 etcd + 若干 worker 节点部署方案进行分析。Kubernetes 的多 master 节点高可用部署可参考[官方文档](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/high-availability/)。
+请参考 K8s [官方文档](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/high-availability/) 来部署一套多 Master 节点高可用集群。
```
Comment (ran-huang) [inline zh/prerequisites.md:166]:
```suggestion
参考 [Kubernetes 官方文档](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/high-availability/)，部署一套多 Master 节点高可用集群。
```

File: zh/prerequisites.md [hunk 6]
```diff
@@ -16,188 +16,182 @@ category: how-to
...
 
-## Kubernetes 系统资源要求
+K8s Master 节点的配置取决于 K8s 集群中 Node 节点个数，节点数越多，需要的资源也就越多。可根据需要做微调。
```
Comment (ran-huang) [inline zh/prerequisites.md:168]:
```suggestion
Kubernetes Master 节点的配置取决于 Kubernetes 集群中 Node 节点个数，节点数越多，需要的资源也就越多。节点数可根据需要做微调。
```

File: zh/prerequisites.md [hunk 7]
```diff
@@ -16,188 +16,182 @@ category: how-to
...
+    {{< copyable "shell-regular" >}}
+    
+    ```shell
+    echo "KUBELET_EXTRA_ARGS=--root-dir=/data1/kubelet" > /etc/sysconfig/kubelet
+    systemctl restart kubelet
+    ```
+   
+    上面会将 Kubelet 数据目录设置为 `/data1/kubelet`。
+    
+2. 通过 kubelet 设置 [预留资源](https://kubernetes.io/docs/tasks/administer-cluster/reserve-compute-resources/) 来保证机器上的系统进程以及 Kubernetes 的核心进程在工作负载很高的情况下仍然有足够的资源来运行，从而保证整个系统的稳定。
```
Comment (ran-huang) [inline zh/prerequisites.md:192]:
```suggestion
2. 通过 kubelet 设置[预留资源](https://kubernetes.io/docs/tasks/administer-cluster/reserve-compute-resources/)，保证机器上的系统进程以及 Kubernetes 的核心进程在工作负载很高的情况下仍然有足够的资源来运行，从而保证整个系统的稳定。
```

File: zh/prerequisites.md [hunk 8]
```diff
@@ -16,188 +16,182 @@ category: how-to
...
+| :--- | :--- |
+| 1-5 | 1vCPUs 4GB Memory|
+| 6-10 | 2vCPUs 8GB Memory|
+| 11-100 | 4vCPUs 16GB Memory|
+| 101-250 | 8vCPUs 32GB Memory|
+| 251-500 | 16vCPUs 64GB Memory|
+| 501-5000 | 32vCPUs 128GB Memory|
 
-- 如果需要部署 Kubernetes 集群的监控系统, 且监控数据需要落盘，则也需要考虑为 Prometheus 准备一块 SAS 盘，后面日志监控系统也需要大的 SAS 盘，同时考虑到机器采购最好是同构的这一因素，因此每台机器最好有两块大的 SAS 盘。
+安装完 Kubelet 之后：
```
Comment (ran-huang) [inline zh/prerequisites.md:179]:
```suggestion
安装完 Kubelet 之后，执行以下步骤：
```

File: zh/prerequisites.md [hunk 9]
```diff
@@ -16,188 +16,182 @@ category: how-to
...
-    - 做了 RAID5 的 SAS 盘，给 Docker 和 kubelet 做数据盘
-    - 两块 NVMe 盘给 TiKV 实例
-
-从上面的分析来看，要支持 5 套 TiDB 集群容量共需要 7 台物理机，其中 3 台为 master 兼 worker 节点，其余 4 台为 worker 节点，机器配置需求如下：
-
-- master 兼 worker 节点：48C 192GB；2 块 SSD 盘，一块做了 RAID5 的 SAS 盘，三块 NVMe 盘
-- worker 节点：48C 192GB；1 块 SSD 盘，一块做了 RAID5 的 SAS 盘，两块 NVMe 盘
-
-使用上面的机器配置，除去各个组件占用的资源外，还有比较多的预留资源。如果要考虑加监控和日志组件，则可以用同样的方法去规划需要采购的机器类型以及配置。
+请根据 [服务器建议配置](https://pingcap.com/docs-cn/stable/hardware-and-software-requirements/#生产环境) 来规划机器的配置。
```
Comment (ran-huang) [inline zh/prerequisites.md:196]:
```suggestion
请根据[服务器建议配置](https://pingcap.com/docs-cn/stable/hardware-and-software-requirements/#生产环境)来规划机器的配置。
```

File: zh/prerequisites.md [hunk 10]
```diff
@@ -16,188 +16,182 @@ category: how-to
...
 
-另外，在生产环境的使用上尽量不要在 master 节点部署 TiDB 实例，或者尽可能少地部署 TiDB 实例。这里的主要考虑点是网卡带宽，因为 master 节点网卡满负荷工作会影响到 worker 节点和 master 节点之间的心跳信息汇报，导致比较严重的问题。
+另外，在生产环境的使用上尽量不要在 K8s Master 节点部署 TiDB 实例，或者尽可能少地部署 TiDB 实例。这里的主要考虑点是网卡带宽，因为 Master 节点网卡满负荷工作会影响到 Worker 节点和 Master 节点之间的心跳信息汇报，导致比较严重的问题。
```
Comment (ran-huang) [inline zh/prerequisites.md:198]:
```suggestion
另外，在生产环境中，尽量不要在 Kubernetes Master 节点部署 TiDB 实例，或者尽可能少地部署 TiDB 实例。因为网卡带宽的限制，Master 节点网卡满负荷工作会影响到 Worker 节点和 Master 节点之间的心跳信息汇报，导致比较严重的问题。
```

File: zh/prerequisites.md [hunk 11]
```diff
@@ -16,188 +16,182 @@ category: how-to
...
-| net.ipv4.conf.all.rp_filter | 1 |
-| net.ipv4.neigh.default.gc_thresh1 | 80000 |
-| net.ipv4.neigh.default.gc_thresh2 | 90000 |
-| net.ipv4.neigh.default.gc_thresh3 | 100000 |
-| net.bridge.bridge-nf-call-iptables | 1 |
-| net.bridge.bridge-nf-call-arptables | 1 |
-| net.bridge.bridge-nf-call-ip6tables | 1 |
-
-在设置 `net.bridge.bridge-nf-call-*` 这几项参数时，如果选项报错，则可通过如下命令检查是否已经加载该模块：
+建议关闭防火墙：
```
[the remark this answers is not in the document]
Comment (weekface) [reply zh/prerequisites.md:21]: done

File: zh/prerequisites.md [hunk 12]
```diff
@@ -16,188 +16,221 @@ category: how-to
...
+cpupower frequency-set --governor performance
+```
 
-    ```shell
-    systemctl start irqbalance
-    ```
+## Ulimit 设置
 
-+ 方法二：禁用 irqbalance，自定义中断号和 CPU 的绑定关系。详情参见脚本 [set_irq_affinity.sh](https://gist.githubusercontent.com/SaveTheRbtz/8875474/raw/0c6e500e81e161505d9111ac77115a2367180d12/set_irq_affinity.sh)。
+TiDB 集群默认会使用很多文件描述符，工作节点上面的 `ulimit` 设置大于等于 `1048576`：
```
Comment (ran-huang) [inline zh/prerequisites.md:148]: Is this what you mean？
```suggestion
TiDB 集群默认会使用很多文件描述符，需要将工作节点上面的 `ulimit` 设置为大于等于 `1048576`：
```

File: zh/prerequisites.md [hunk 13]
```diff
@@ -16,188 +16,221 @@ category: how-to
...
+EOF
+
+sysctl --system
+```
 
-## ulimit 设置
+## Docker 服务
 
-TiDB 集群默认会使用很多文件描述符，工作节点和上面的 Docker 进程的 `ulimit` 必须设置大于等于 `1048576`：
+安装 Docker 时，建议选择 Docker CE 18.09.6 及以上版本。 请参考 [Docker 安装指南](https://docs.docker.com/engine/install/centos/) 进行安装。
```
Comment (ran-huang) [inline zh/prerequisites.md:162]:
```suggestion
安装 Docker 时，建议选择 Docker CE 18.09.6 及以上版本。请参考 [Docker 安装指南](https://docs.docker.com/engine/install/centos/) 进行安装。
```

File: zh/prerequisites.md [hunk 14]
```diff
@@ -16,188 +16,221 @@ category: how-to
...
 
-对于服务器 disk、memory、CPU 的选择要根据对集群的容量规划以及部署拓扑来定。线上 Kubernetes 集群部署为了保证高可用，一般需要部署三个 master 节点、三个 etcd 节点以及若干个 worker 节点。同时，为了充分利用机器资源，master 节点一般也充当 worker 节点（也就是 master 节点上也可以调度负载）。通过 kubelet 设置[预留资源](https://kubernetes.io/docs/tasks/administer-cluster/reserve-compute-resources/)来保证机器上的系统进程以及 Kubernetes 的核心进程在工作负载很高的情况下仍然有足够的资源来运行，从而保证整个系统的稳定。
+Kubernetes Master 节点的配置取决于 Kubernetes 集群中 Node 节点个数，节点数越多，需要的资源也就越多。节点数可根据需要做微调。
 
-下面按 3 Kubernetes master + 3 etcd + 若干 worker 节点部署方案进行分析。Kubernetes 的多 master 节点高可用部署可参考[官方文档](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/high-availability/)。
-
-## Kubernetes 系统资源要求
-
-- 每台机器需要一块比较大的 SAS 盘（至少 1T），这块盘用来存 Docker 和 kubelet 的数据目录。Docker 的数据主要包括镜像和容器日志数据，kubelet 主要占盘的数据是 [emptyDir](https://kubernetes.io/docs/concepts/storage/volumes/#emptydir) 所使用的数据。
+| K8s 集群 Node 节点个数 | K8s Master 节点配置 |
```
Comment (ran-huang) [inline zh/prerequisites.md:208]:
```suggestion
| Kubernetes 集群 Node 节点个数 | Kubernetes Master 节点配置 |
```

**Pull request merged**